helm/cloud-provider-vsphere/charts/vsphere-csi-driver/Chart.yaml

```diff
@@ -1,12 +1,14 @@
 apiVersion: v2
 name: vsphere-csi-driver
-version: 1.0.0
+appVersion: 3.2.0
+version: 3.2.0
```

Comment:
Normally, `appVersion` refers to the upstream app version, and `version` refers to the chart version. Since we created the chart here, we set it as `1.0.0`. I am not sure if we always keep them the same but  I am OK for now.

Reply:
I'm not sure either but I don't really see the point of having a different chart version.